components/constants/pref_names.cc

```diff
@@ -46,6 +46,7 @@ const char kReferralHeaders[] = "brave.referral.headers";
 const char kReferralAndroidFirstRunTimestamp[] =
     "brave.referral_android_first_run_timestamp";
 const char kNoScriptControlType[] = "brave.no_script_default";
+const char kFingerprintingControlType[] = "brave.fingerprinting";
```

Comment:
wait, why this is a preference, but not a content settings value?
can this be changed per website? If yes, then it should be stored as a "default" value for a content settings.

Reply:
In order to create a toggle switch (instead of a drop-down menu) it was necessary to create a pref for the global setting. But we are still tying both the global and per-site content settings to the respective toggle switches.

Reply:
> In order to create a toggle switch (instead of a drop-down menu) for the global setting, it was necessary to create a pref

You can create it with an empty pref. Please take a look at how `forgetFirstPartyStorageControlType` is implemented. It uses an empty pref and handles changes via callback.